src/lib/api/calendars.ts

```diff
@@ -126,6 +151,11 @@ export async function process(message: ApplicationMessage) {
   switch (type) {
     case "calendar_created":
       return await onCalendarCreated(meta, data);
+    case "calendar_access_requested":
+      // @TODO: store calendar access request
+      return;
+    case "calendar_access_accepted":
+      return await onCalendarAccessAccepted(meta, data);
```

Comment:
@ckjr3000 @antiantivirus in order to process these events we to store access requests we receive, and flip a flag somewhere when we are given access to a calendar. Is there already a place for these? Would appreciate any tips on implementation :+1: